examples/Makefile-posix

```diff
@@ -74,6 +74,10 @@ configure_OPTIONS                   = \
     --with-examples=posix             \
     $(NULL)
 
+ifeq ($(PLATFORM_UDP), 1)
```

Comment:
Is this intentionally specific to `Makefile-posix`?

Reply:
Yes, I think this should be put into *common-switches.mk*. Thanks!